src/content/en/2020/privacy.md

```diff
@@ -12,12 +12,166 @@ translators: []
+  caption="Top 10 Trackers",
+  description="This figure shows the prevalence of the 10 most popular trackers used on mobile and desktop clients.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQJMtHv0Y1JcQJkcVyrqBk9dsujZeDtOZEv7uvE0xM2VrQSuTUDFya41TeRlTZDDe2rWmHwDghW3Dev/pubchart?oid=850649042&format=interactive",
+  sheets_gid="1677398038"
+  )
+}}
+
+The largest player on the online tracking market is definitely Google, with eight of  its tracking domains present in the top 10 trackers and prevalent on at least 70% of websites. Following are Facebook and Cloudflare. 
+
+WhoTracksMe’s tracker list also defines categories that the trackers belong to. The following figure shows the distribution of the different categories for the 100 largest trackers.
+
+{{ figure_markup(
+  image="tracker_categories.png",
+  caption="Categories of the 100 most popular trackers",
+  description="This figure shows distribution of the top 100 trackers on the web.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQJMtHv0Y1JcQJkcVyrqBk9dsujZeDtOZEv7uvE0xM2VrQSuTUDFya41TeRlTZDDe2rWmHwDghW3Dev/pubchart?oid=1117413918&format=interactive",
+  sheets_gid="1431872451",
+  )
+}}
+
+Nearly 60% of the most popular trackers are advertising-related. This could be due to the profitability of the online advertising market.
+
+### Cookies
+We looked into the most popular cookies being set on websites in HTTP's response header, according to their name and domain.
+
+<figure markdown>
+| Client | Domain | Cookie Name | Websites present on   |
+| -------- | ------- | ------ | ------ |
+| desktop |	doubleclick.net|	test_cookie|	24%|
+|desktop|	facebook.com|	fr|		10%|
+|desktop|	youtube.com|	VISITOR_INFO1_LIVE|	10%
+|desktop|	youtube.com|	YSC|10%
+|desktop|	doubleclick.net|	IDE|9%
+|desktop|	doubleclick.net|unknown|	9%
+|desktop|	youtube.com	|GPS|9%
+|desktop|	doubleclick.net|	unknown|	8%
+|desktop|	google.com|	NID|	6%
+|desktop|	doubleclick.net|	unknown|6%
+|mobile|	doubleclick.net|	test_cookie|32%
+|mobile|	doubleclick.net	|IDE|	21%
+|mobile|	facebook.com|	fr|	10%
+|mobile|	youtube.com|	VISITOR_INFO1_LIVE|10%
+|mobile	|youtube.com|	YSC|10%
+|mobile|	google.com|	NID|10%
+|mobile|	youtube.com	|GPS|	8%
+|mobile|	doubleclick.net	|DSID|	7%
+|mobile|	yandex.ru|	yandexuid|	6%
+|mobile|	yandex.ru|	i|	6%
+
+<figcaption>{{ figure_link(caption="Top cookies", sheets_gid="732942035", sql_file="top100_cookies_set_from_header.sql") }}</figcaption>
+</figure>
+
+As found previously, Google’s  tracking domain ‘doubleclick.net’  sets cookies on roughly one fourth of websites on a mobile client and one third of all  websites on a desktop client. Again, nine out of the ten most popular cookies on desktop client and 7 on mobile are set by a Google domain. This is a lower bound for the amount of websites the cookie is set on, since we are only conting cookies set via an HTTP header, a large amount of tracking cookies is set by using third-party scripts.
```

Comment:
```suggestion
As found previously, Google’s tracking domain ‘doubleclick.net’ sets cookies on roughly a quarter of websites on a mobile client and one third of websites on a desktop client. Again, nine out of the ten of the most popular cookies on desktop client and 7 on mobile are set by a Google domain. This is a lower bound for the amount of websites the cookie is set on, since we are only counting cookies set via an HTTP header, a large amount of tracking cookies are set by using third-party scripts.
```